Automattic/jetpack
# General: Move docker ignored files before the list of files we are not going to ignore

Fixes a weirdness in one of the release bundling steps.

#### Changes proposed in this Pull Request:

* Moves some ignored files in `.gitignore` a little be upper because the `build-release-branch.sh` script will stop parsing lines as ignored as soon as it finds a line with the comment: `## Things we will need in release branches`

#### Testing instructions:



* Check this PR. confirm that all previously ignored files (lines moved in this changeset) are still gitignored.


#### Proposed changelog entry for your changes:
None needed
